### ISSUE TITLE
Bug: Community Lovecraft Mythos has disapperared from the results

### ISSUE THREAD

**Issue by Zep251** [tgxn/lemmy-explorer#191]
**Describe the bug**
The community [!lovecraft_mythos@lemmy.world](https://lemmy.world/c/lovecraft_mythos) won't appear anymore in the results. If i search the exact name instead appear another community [!horrorlit@lemmy.world](https://lemmy.world/c/horrorlit)

It also don't appear with any other key like "horror" or "lovecraft"

searcn link: https://lemmyverse.net/communities?query=Lovecraft+Mythos

I ask you to please fix this issue.

![348](https://github.com/user-attachments/assets/0226a2fd-842e-45ba-9022-9481b824d1d9)
![5430](https://github.com/user-attachments/assets/11ef5b30-08b2-48bb-b265-92ff61aec2cb)
![451](https://github.com/user-attachments/assets/23ff57ee-2bef-4898-a6b9-e567dfc10be5)

**To Reproduce**
1. Go to 'https://lemmyverse.net/communities'
2. Type in the search bar  'Lovecraft Mythos' or 'horror' or 'Lovecraft'
3. Scroll down to the results don't show the group 

**Expected behavior**
Showing the active group https://lemmy.world/c/lovecraft_mythos in the search results.

**Additional context**
I ask you to please fix this issue.

**Comment by tgxn** [comment]
yeah I've been told there seems to be a few missing, having a look tonight

**Comment by tgxn** [comment]
![image](https://github.com/user-attachments/assets/1bafdcc0-86c2-4bd6-bd3e-d5dca9cce833)

https://lemmyverse.net/communities?query=Lovecraft+Mythos

It's back :)

**Comment by tgxn** [comment]
It's gone again :S 

I don't understand why it's this community that continually goes missing, while the others from lemmy.world are coming through.

**Comment by tgxn** [comment]
just over 10.5k communities, lemmy.world is truly massive, and the 50 per-page limit is nasty

```
[CommunityList] [lemmy.world] Page 210, Results: 50
[CommunityList] [lemmy.world] Page 211, Fetching...
[CommunityList] [lemmy.world] Page 211, Results: 48
[CommunityList] [lemmy.world] Page 212, Fetching...
[CommunityList] [lemmy.world] Page 212, Results: 0
[CommunityList] [lemmy.world] Ended Success (10548 results)
```

for now I'm thinking of tracking how long each run takes to see if

**Comment by tgxn** [comment]
have done a lot of refactoring and am running on #192 for the crawler side, it's still there for now.

```
[CommunityList] [lemmy.world] Page 211, Results: 50
[CommunityList] [lemmy.world] Page 212, Fetching...
[CommunityList] [lemmy.world] Page 212, Results: 11
[CommunityList] [lemmy.world] Page 213, Fetching...
[CommunityList] [lemmy.world] Page 213, Results: 0
[CommunityList] [lemmy.world] Ended Success (10561 results)
[Community] [lemmy.world] Finished in 571.277s
```

crawls for `lemmy.world` take almost 10 minutes..

I've also added an output validator that just checks `lovecraft_mythos` is present, please don't delete it :D
